fixtures/lightmaxx/vega-zoom-wash.json

```diff
@@ -28,205 +28,252 @@
+      }
     },
     "Color Temperature": {
-      "type": "Color Temperature"
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "cold",
+        "colorTemperatureEnd": "warm",
+        "helpWanted": "Correct direction?"
+      }
     },
     "Color Macro": {
-      "type": "Multi-Color",
       "capabilities": [
         {
-          "range": [0, 10],
-          "name": "open"
+          "dmxRange": [0, 10],
+          "type": "Nothing"
         },
         {
-          "range": [11, 20],
-          "name": "red",
-          "color": "#ff0000"
+          "dmxRange": [11, 20],
+          "type": "ColorPreset",
+          "colors": ["Red"]
         },
         {
-          "range": [21, 30],
-          "name": "green",
-          "color": "#00ff00"
+          "dmxRange": [21, 30],
+          "type": "ColorPreset",
+          "colors": ["Green"]
         },
         {
-          "range": [31, 40],
-          "name": "blue",
-          "color": "#0000ff"
+          "dmxRange": [31, 40],
+          "type": "ColorPreset",
+          "colors": ["Blue"]
         },
         {
-          "range": [41, 50],
-          "name": "white",
-          "color": "#ffffff"
+          "dmxRange": [41, 50],
+          "type": "ColorPreset",
+          "colors": ["White"]
         },
         {
-          "range": [51, 60],
-          "name": "yellow",
-          "color": "#ffff00"
+          "dmxRange": [51, 60],
+          "type": "ColorPreset",
+          "colors": ["Yellow"]
         },
         {
-          "range": [61, 70],
-          "name": "cyan",
-          "color": "#00ffff"
+          "dmxRange": [61, 70],
+          "type": "ColorPreset",
+          "colors": ["Cyan"]
         },
         {
-          "range": [71, 80],
-          "name": "lightblue",
-          "color": "#64c8ff"
+          "dmxRange": [71, 80],
+          "type": "ColorPreset",
+          "colors": ["Lightblue"]
         },
         {
-          "range": [81, 90],
-          "name": "pink",
-          "color": "#ff00ff"
+          "dmxRange": [81, 90],
+          "type": "ColorPreset",
+          "colors": ["Pink"]
         },
         {
-          "range": [91, 100],
-          "name": "lightgreen",
-          "color": "#7fff3c"
+          "dmxRange": [91, 100],
+          "type": "ColorPreset",
+          "colors": ["Lightgreen"]
         },
         {
-          "range": [101, 110],
-          "name": "lightred",
-          "color": "#ff7f7f"
+          "dmxRange": [101, 110],
+          "type": "ColorPreset",
+          "colors": ["Lightred"]
         },
         {
-          "range": [111, 120],
-          "name": "RGB"
+          "dmxRange": [111, 120],
+          "type": "ColorPreset",
+          "colors": ["White"],
+          "comment": "RGB"
         },
         {
-          "range": [121, 130],
-          "name": "RGW"
+          "dmxRange": [121, 130],
+          "type": "ColorPreset",
+          "colors": ["Lightyellow"]
```

Comment:
Keep "RGW" as comment for clarification (so it'll be easier to add color component mixing information in #437)